code/client/src/sdk/mafia/ui/c_game_gui_2_module.h

```diff
@@ -8,6 +8,8 @@
 #include "support/c_fader.h"
 
 #include "../../ue/c_variant.h"
+#include "sdk/ue/sys/sodb/c_database_interface.h"
+#include "sdk/ue/c_weak_ptr.h"
```

Comment:
In every SDK files, we have to use relative imports

Reply:
Could you elaborate?

Reply:
See https://github.com/MafiaHub/MafiaMP/pull/100#discussion_r1515155080